tracing/tracing.go

```diff
@@ -18,9 +18,10 @@ import (
 	semconv "go.opentelemetry.io/otel/semconv/v1.21.0"
 	"go.opentelemetry.io/otel/trace"
 
-	"github.com/mailgun/holster/v4/errors"
 	"github.com/sirupsen/logrus"
 	"github.com/uptrace/opentelemetry-go-extra/otellogrus"
+
```

Comment:
nit: Should revert this unnecessary change.

Reply:
ah thanks, that's my IDE formatting imports, will do

Reply:
Oh, I see.  It placed organizational imports in a 3rd section.  That's a new thing with Golang.  So it's not a mistake.

Reply:
Yeah, but there are 4 sections now.